draft-ietf-ppm-dap.md

```diff
@@ -1479,36 +1487,58 @@ struct {
 } InputShareAad;
 ~~~
 
+~~~ tls-presentation
+struct {
+ReportID id;
+ReportError error;
+} ReportUploadStatus;
+
+struct {
+ReportUploadStatus status<0..2^32-1>;
+} UploadResponse;
+~~~
+
 If the upload request is malformed, the Leader aborts with error
 `invalidMessage`.
 
 If the Leader does not recognize the task ID, then it aborts with error
 `unrecognizedTask`.
 
+Otherwise, the Leader responds with HTTP status 200, a body consisting
+of `UploadResponse` and the media type `application/dap-report-resp`. If
```

Comment:
Per RFC 9205, we don't prescribe specific HTTP statuses, but leave it up to implementations to choose which success status to use.
```suggestion
Otherwise, the Leader responds with a body consisting
of an `UploadResponse` and the media type `application/dap-report-resp`. If
```